### PR TITLE
DOCSP-42020: queues feedback

### PR DESCRIPTION
**Pull request by rustagir**
https://jira.mongodb.org/browse/DOCSP-42020

Replaces `collection()` with `table()` 
and removes instructions to explicitly register service providers.

[Staging](https://deploy-preview-152--docs-laravel.netlify.app/queues/)

### Checklist

- [ ] Add tests and ensure they pass

### CODE REVIEW

File: docs/queues.txt [hunk 1]
```diff
@@ -11,6 +11,16 @@ Queues
 .. meta::
    :keywords: php framework, odm, code example, jobs
 
+Overview
+--------
+
+In this guide, you can learn how to use MongoDB as your database for
+Laravel Queue. Laravel Queue allows you to create queued jobs that can
+processed in the background.
```
Comment (jordan-smith721) [inline docs/queues.txt]:
```suggestion
Laravel Queue. Laravel Queue allows you to create queued jobs that are
processed in the background.
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM + 1 fix

**Review by GromNaN — APPROVED** [review]

File: docs/queues.txt [hunk 2]
```diff
@@ -91,16 +101,13 @@ how to handle failed jobs:
...
        jobs. The value is ``failed_jobs`` by default.
 
-Then, add the service provider in your application's
-``config/app.php`` file:
-
-.. code-block:: php
-
-   MongoDB\Laravel\MongoDBQueueServiceProvider::class,
+The {+odm-short+} automatically provides the
+``MongoDB\Laravel\MongoDBQueueServiceProvider::class`` class as the
```
Comment (bisht2050) [inline docs/queues.txt:109]: @GromNaN  Isn't `MongoDBQueueServiceProvider` completely removed (as in it doesn't exist anymore) ?

Comment (bisht2050) [reply docs/queues.txt:109]: Please disregard this comment, I missed to see the target branch.

**Review by norareidy — APPROVED** [review]

**Pull request merged**